dependency-injection/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'java'
```

Comment:
I haven't looked at the code in detail, but yes - basically, most projects here use Maven, so let's make sure this does the same.